plugins/meta/tuning/tuning.go

```diff
@@ -205,6 +332,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 			}
 		}
 
+		isTuned := (tuningConf.Mac != "" || tuningConf.Mtu != 0 || tuningConf.Promisc)
+		if err = createBackup(args.IfName, args.ContainerID, tuningConf.DataDir, isTuned, tuningConf); err != nil {
```

Comment:
could we just skip creating a backup if nothing was tuned?

Reply:
I initially though that it would be better to create a file even if nothing's changed just to avoid using the file existence check as indicator of whenever something has to be restored. However changed that as it seems that it may be more clear in general.